presto-hive/src/test/java/io/prestosql/plugin/hive/TestBackgroundHiveSplitLoader.java

```diff
@@ -443,14 +448,15 @@ public void testFullAcidTableWithOriginalFilesFails()
 
         List<String> filePaths = ImmutableList.of(
                 tablePath + "/000000_1",
+                tablePath + "/delta_0000002_0000002_0000/_orc_acid_version",
                 tablePath + "/delta_0000002_0000002_0000/bucket_00000");
 
         for (String path : filePaths) {
             File file = new File(path);
             if (!file.getParent().equals(tablePath.toString())) {
-                assertTrue(file.getParentFile().mkdirs(), "Failed creating directory " + file.getParentFile());
+                assertTrue(file.getParentFile().exists() || file.getParentFile().mkdirs(), "Failed creating directory " + file.getParentFile());
```

Comment:
this allows you to remove the `if (!file.getParent().equals(tablePath.toString()))` line above

Reply:
fixed.